gnucash/report/CMakeLists.txt

```diff
@@ -17,9 +17,21 @@ set (report_SOURCES
   gnc-report.c
 )
 
+set (qrencode_HEADERS gnc-qrencode.h)
+
+if (HAVE_QRENCODE)
+  set (qrencode_SOURCES gnc-qrencode.c)
+  set (qrencode_LIBRARY PkgConfig::QRENCODE)
+else ()
+  set (qrencode_SOURCES gnc-qrencode-stub.c)
+  set (qrencode_LIBRARY)
+endif ()
+
```

Comment:
You need to add the new files to the `local_dist` list at the bottom so that they're included in the tarball. Since qrencode_SOURCES is variable you have to name the two files so that they're both added.